bolt/onboarding.go

```diff
@@ -118,13 +118,12 @@ func (c *Client) Generate(ctx context.Context, req *platform.OnboardingRequest)
 		return nil, err
 	}
 
-	perms := platform.OperPermissions()
-	perms = append(perms, platform.OrgAdminPermissions(o.ID)...)
-	writeBucketPerm, err := platform.NewPermissionAtID(bucket.ID, platform.WriteAction, platform.BucketsResource)
+	perms := platform.OperPermissions(o.ID)
+	writeBucketPerm, err := platform.NewPermissionAtID(bucket.ID, platform.WriteAction, platform.BucketsResource, bucket.OrganizationID)
```

Comment:
Shouldn't the onboarding token be for all buckets rather than just a single bucket?

Reply:
yup. It is. Dunno why I left this here.